### PR TITLE
add homepage

### PR DESCRIPTION
**Pull request by Corcit**

### CODE REVIEW

File: src/views/index.ejs [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+            <nav class="menu-block" id="append-menu-header">
+              <div class="mobile-menu-head">
+                <div class="go-back">
+                  <i class="fa fa-angle-left"></i>
+                </div>
+                <div class="current-menu-title"></div>
+                <div class="mobile-menu-close">&times;</div>
+              </div>
+              <ul class="site-menu-main">
+                <li class="nav-item">
```
Comment (felmez) [inline src/views/index.ejs:61]: - here we can add more navbar items like => API for api docs link and => Tech used to go for the tech section

image

File: src/views/index.ejs [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+              </div>
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="1000" data-aos-once="true">
+                <img src="image/l4/brand-logo-3.svg" alt="">
+              </div>
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="1200" data-aos-once="true">
+                <img src="image/l4/brand-logo-4.svg" alt="">
+              </div>
+            </div>
+          </div>
+        </div>
```
Comment (felmez) [inline src/views/index.ejs:136]: - here we need to duplicate this section and convert it to technologies used in our api and insert these tech logos like nodejs, express,mongodb etc

image

**Review by felmez — CHANGES_REQUESTED** [review]
good job

**Comment by halitbatur** [comment]
@Corcit status update on the changes requested?

File: src/views/index.ejs (hunk 1 above)
Comment (Corcit) [reply src/views/index.ejs:61]: I have added the tech part but what should be the end point for API.

File: src/views/index.ejs (hunk 2 above)
Comment (Corcit) [reply src/views/index.ejs:140]: Done ✔️

File: src/views/index.ejs (hunk 1 above)
Comment (felmez) [reply src/views/index.ejs:61]: api link should only redirect to out /api-docs route thats it

**Comment by halitbatur** [comment]
@yaman3bd could you jump to this one and finish it?

**Comment by halitbatur** [comment]
@yaman3bd this needs to get merged with updated main branch and fix conflicts and then its ready to be merged

**Review by halitbatur — APPROVED** [review]

**Review by felmez — APPROVED** [review]

**Pull request merged**